pkg/logging/logging_test.go

```diff
@@ -357,10 +357,10 @@ func TestNewContext(t *testing.T) {
 	ctx := WithFields(nil)
 	require.NotNil(t, ctx)
 
-	ctx = WithFields(nil, "hello")
+	ctx = WithFields(ctx, slog.Attr{})
```

Comment:
Took the liberty of adding two other trivial tests for the two removed ones.

Reply:
Great! Well spotted.